### ISSUE TITLE
Duo 0.11.1 breaks make

### ISSUE THREAD

**Issue by untidy-hair** [recurly/recurly-js#197]
I just noticed they released duo 0.11.1 and it breaks 'make' tasks :)
```bash
$ make
Error: unrecognized flag --harmony-generators
Try --help for options

/path/to/recurly-js/node_modules/duo/lib/duo.js:373
Duo.prototype.cleanCache = function *() {
                                    ^
SyntaxError: Unexpected token *
    at Module._compile (module.js:439:25)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Duo (/path/to/recurly-js/node_modules/duo/bin/_duo:511:13)
    at create (/path/to/recurly-js/node_modules/duo/bin/_duo:287:13)
    at print (/path/to/recurly-js/node_modules/duo/bin/_duo:223:13)
    at Object.<anonymous> (/path/to/recurly-js/node_modules/duo/bin/_duo:185:44)
make: *** [build] Error 8
```
Updating package.json with 0.10.x fixes this but I don't know if that is the right choice
```
-    "duo": "0.x",
+    "duo": "0.10.x",
```
Thanks.

**Comment by chrissrogers** [comment]
I suspect this is due to using a node version before 0.11.x -- which node version are you running?

**Comment by chrissrogers** [comment]
We experienced the same issue with Travis, which was on node 0.10.x. https://travis-ci.org/recurly/recurly-js/builds/56494417 shows the fix (updating node to 0.12.x) applied.

**Comment by untidy-hair** [comment]
@chrissrogers Thanks for the clarification.
